Backpack/Dialog/Classes/BPKDialogIconView.m

```diff
@@ -49,19 +49,16 @@ - (CGSize)intrinsicContentSize {
     return [[self class] viewSize];
 }
 
-- (void)setIconBackgroundColor:(UIColor *_Nullable)iconBackgroundColor {
-    self.iconContainerView.backgroundColor = iconBackgroundColor;
+- (void)setIconDefinition:(BPKDialogIconDefinition *)iconDefinition {
+    self.iconContainerView.backgroundColor = iconDefinition.iconBackgroundColor;
```

Comment:
This needs to update the local variable `_iconDefintion` too and it should have the regular  

```objective-c
if (iconDefintion != _iconDefintion) {
}
```

guards